.node-version

```diff
@@ -1 +1 @@
-20.10.0
+21.0.0
```

Comment:
should we change this version also for workflows? I mean these 2:
- https://github.com/latina-in-tech/latina-in-tech.github.io/blob/main/.github/workflows/build-and-deploy.yml#L13
- https://github.com/latina-in-tech/latina-in-tech.github.io/blob/main/.github/workflows/verify-code.yml#L15